ci/nightly-test.sh

```diff
@@ -117,7 +117,6 @@ start_run_teardown "emergency_upgrade_test.sh"
 start_run_teardown "emergency_upgrade_test_balances.sh"
 start_run_teardown "upgrade_after_emergency_upgrade_test.sh"
 start_run_teardown "sync_test.sh timeout=500"
-start_run_teardown "gov96.sh"
```

Comment:
It seems like we can also remove the `itst02.sh` test, which is similar to `gov96.sh`.